docs/user-guide/integrations/sigfox-example.md

```diff
@@ -0,0 +1,226 @@
+---
+layout: docwithnav
+title: Get device parameters using SigFox integration
+description: SigFox integration guide
+
+---
+
+{% assign feature = "Platform Integrations" %}{% include templates/pe-feature-banner.md %}
+
+This tutorial will show how to get attribute value of a device using SigFox integration.
```

Comment:
Will show how to push downlink messages to devices connected via Sigfox integration when user updates device attribute using ThingsBoard UI